.github/workflows/trigger-website-build.yml

```
@@ -0,0 +1,22 @@
+  push:
+    paths:
+      - 'docs/**'
+  workflow_dispatch:
+
+jobs:
+  trigger-website:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Trigger website build
+      run: |
```

actionlint (1.7.4)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)